certbot/cli.py

```diff
@@ -366,33 +367,40 @@ def _get_help_string(self, action):
     }),
     ("register", {
         "short": "Register for account with Let's Encrypt / other ACME server",
-        "opts": "Options for account registration & modification"
+        "opts": "Options for account registration & modification",
+        "usage": "\n\n  certbot register --email user@example.com [options]"
     }),
     ("unregister", {
         "short": "Irrevocably deactivate your account",
-        "opts": "Options for account deactivation."
+        "opts": "Options for account deactivation.",
+        "usage": "\n\n  certbot unregister --email user@example.com [options]"
```

Comment:
Although it probably should work, using `--email` here has no effect. Instead, I think this should just be `certbot unregister [options]`.